package.json

```diff
@@ -69,6 +69,7 @@
     "postinstall": "opencollective || exit 0"
   },
   "dependencies": {
+    "@applitools/eyes-cypress": "^3.6.15",
```

Comment:
I'm not sure this should be in `dependencies`, but rather should be `devDependencies`, since it is not required to install BootstrapVue into a project.

Reply:
My mistake. Fixed. Thank you!